gallery/src/styles/NxCard/NxCardPage.tsx

```diff
@@ -8,21 +8,22 @@ import React from 'react';
 
 import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
 
-import NxCardRowLayoutExample from './NxCardRowLayoutExample';
-import NxCardColumnLayoutExample from './NxCardColumnLayoutExample';
-import NxCardRowAltLayoutExample from './NxCardRowAltLayoutExample';
+import './NxCardPageExamples.scss';
 
-const nxCardRowLayoutCode = require('!!raw-loader!./NxCardRowLayoutExample').default,
-    nxCardColumnLayoutCode = require('!!raw-loader!./NxCardColumnLayoutExample').default,
-    nxCardRowAltLayoutCode = require('!!raw-loader!./NxCardRowAltLayoutExample').default;
+import NxCardLayoutExample from './NxCardLayoutExample';
+
+const nxCardLayoutCode = require('!!raw-loader!./NxCardLayoutExample').default,
+    NxCardCustomSCSSCode = require('!!raw-loader!./NxCardPageExamples.scss').default;
 
 const NxCardPage = () =>
   <>
     <GalleryDescriptionTile>
       <p className="nx-p">
-        Cards can be displayed in two orientations: rows and columns. Cards can be transformed between rows and columns
-        with a simple class change on their container. There is also a row variant supplied which displays the call out
-        and content side-by-side, this variant cannot be transformed into column oriented cards.
+        Cards can be displayed in two orientations: vertically or horizontally. At its most basic a card is simply a
```

Comment:
Various places in the docs need to be updated to remove horizontal card and column container mentions

Reply:
> Various places in the docs need to be updated to remove horizontal card and column container mentions

I only found this one (stupidly obvious) instance that I missed: https://github.com/sonatype/sonatype-react-shared-components/pull/307/commits/cccad606804a64d48ec59e77a1a47db856ef371f

Reply:
> I only found this one (stupidly obvious) instance that I missed: cccad60

I thought I had seen one in the class table but maybe you already got it, can't find it now.